dist/index.js

```diff
@@ -3006,7 +3006,7 @@ class Octokit {
       error: console.error.bind(console)
     }, options.log);
     this.hook = hook; // (1) If neither `options.authStrategy` nor `options.auth` are set, the `octokit` instance
-    //     is unauthenticated. The `this.auth()` method is a no-op and no request hook is registred.
```

Comment:
For my reference, this appears to come from here; https://github.com/octokit/graphql-action/blob/5c1fea5240db8062f5a757c9c3d9a200ab82d970/dist/index.js#L3023
originating here:
https://github.com/octokit/core.js/blob/4d96857fe87c57330fe024a2b0dbf7f1848cd769/src/index.ts#L123